deploy/helm/scf/assets/operations/instance_groups/api.yaml

```diff
@@ -170,12 +170,56 @@
       internal: 9024
     run:
       healthcheck:
+        ccng_monit_http_healthcheck:
+          readiness:
+            # This job exists just to healthcheck cloud_controller_ng; we're
+            # already doing that separately.
+            exec:
+              command: [/bin/true]
+          liveness:
+            exec:
+              command:
+              - /usr/bin/pgrep
+              - --full
+              - --exact
+              - bash /var/vcap/jobs/cloud_controller_ng/bin/ccng_monit_http_healthcheck
         cloud_controller_ng:
           readiness: &cloud_controller_ng_readiness
+            httpGet:
+              path: /healthz
+              port: 9022
+          # We don't want a liveness probe here as we do migration here, and we
+          # do not want to interrupt that.
+          #liveness:
+          #  exec:
+          #    command:
+          #    - /usr/bin/pgrep
+          #    - --full
+          #    - /var/vcap/packages/cloud_controller_ng/cloud_controller_ng/bin/cloud_controller
+        local_worker_1:
+          readiness: &cc_local_worker_readiness
             exec:
-              command: [curl, --fail, --head, --silent, http://127.0.0.1:9022/healthz]
+              command: [/usr/bin/pgrep, --full, cc_api_worker]
```

Comment:
This isn't awesome, but there isn't a good health endpoint for the worker (nor the separate cc-worker job).